UnityProject/Assets/Scripts/UpdateTextures.cs

```diff
@@ -124,6 +125,15 @@ public void DO()
 			        }
 		        }
 	        }
+
+	        var piss = component.GetComponentsInChildren<TMP_InputField>();
```

Comment:
might be worth making it a separate script or an editor tool

Reply:
fixed